package.json

```diff
@@ -70,7 +70,7 @@
     "qunit-dom": "^1.6.0"
   },
   "engines": {
-    "node": "12.* || 14.*"
+    "node": "12.* || >= 14"
```

Comment:
Maybe `12.* || 14.* || >= 16`?
Node.js 15 is no longer supported.

Reply:
great idea!